docs/spec/api.md

```diff
@@ -117,6 +117,9 @@ specification to correspond with the versions enumerated here.
 			<li>Deleting a manifest by tag has been deprecated.</li>
 			<li>Specified `Docker-Content-Digest` header for appropriate entities.</li>
 			<li>Added error code for unsupported operations.</li>
+			<li>Added capability of doing streaming upload to PATCH blob upload.</li>
```

Comment:
I don't want to bike shed over this so I will offer my opinion and make the change you suggested. My opinion is the API version should be released with the reference implementation otherwise implementations of intermediate versions may pop up and we would either need to support them or document their lack of support.